lib/Dialect/Arc/CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ add_circt_dialect_library(CIRCTArc
 
   DEPENDS
   MLIRArcIncGen
+  MLIRHWIncGen
+  MLIRSeqIncGen
```

Comment:
Not sure if this is what breaks the build, but including the whole `HWOps.h` and `SeqOps.h` but only linking against the tablegen generated files looks a bit dangerous to me.

Reply:
You're right :grin: